test/e2e/specs/editor/blocks/table.spec.js

```diff
@@ -0,0 +1,282 @@
+/**
+ * WordPress dependencies
+ */
+const { test, expect } = require( '@wordpress/e2e-test-utils-playwright' );
+
+test.describe( 'Table', () => {
+	test.beforeEach( async ( { admin } ) => {
+		await admin.createNewPost();
+	} );
+
+	test( 'displays a form for choosing the row and column count of the table', async ( {
+		editor,
+		page,
+	} ) => {
+		await editor.insertBlock( { name: 'core/table' } );
+
+		// Check for existence of the column count field.
+		const columnCountLabel = page.locator(
+			'role=spinbutton[name="Column count"i]'
+		);
+		await expect( columnCountLabel ).toHaveCount( 1 );
+
+		// Modify the column count.
+		await columnCountLabel.click();
+		const currentColumnCount = await page.evaluate(
+			() => document.activeElement.value
+		);
+		expect( currentColumnCount ).toBe( '2' );
```

Comment:
Playwright has a `toHaveValue` assertion that makes the call to `page.evaluate` unnecessary.